cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -154,3 +153,50 @@ func triggerPull(repositoryName string, pullConnectionDetails abaputils.Connecti
 	uriConnectionDetails.URL = body.Metadata.URI + expandLog
 	return uriConnectionDetails, nil
 }
+
+func checkPullRepositoryConfiguration(options abapEnvironmentPullGitRepoOptions) error {
+	if len(options.RepositoryNames) > 0 && len(options.Repositories) > 0 {
+		log.Entry().Info("It seems like you have specified both the repositories to be pulled as an in-line configuration as well as in the dedicated repositories configuration file.")
+		log.Entry().Info("Please note that in this case the dedicated repositories configuration file will be handled with priority.")
+	}
+	if len(options.RepositoryNames) == 0 && len(options.Repositories) == 0 {
```

Comment:
options.Repositories is a string, isn't it?

Reply:
true, that was also a leftover from the old implementation.